### PR TITLE
Defect/479 accessibility

### PR DESCRIPTION
**Pull request by esotrope**
new tab focus highlighting and hover contrast

image

image

image

image

image

image

image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html [hunk 1]
```diff
@@ -8,17 +8,17 @@
       [label]="assessmentExam.assessment.label"
       [gradeCode]="assessmentExam.assessment.grade"
     ></assessment-label>
-
-    <sb-radio-group
+    <sb-radio-button-group
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: the newer component highlights correctly when tabbed to

File: webapp/src/main/webapp/src/app/exam/component/show-results-divider/show-results-divider.component.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <hr />
 <a
+  href="javascript:void(0)"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/exam/component/show-results-divider/show-results-divider.component.html]: cant receive focus without this

**Comment by esotrope** [comment]
@malaffoon something to note is that advanced filters are not tab accessible or at least not easily tab accessible. I don't want to touch that unless i have the license for a full redesign of it though

**Comment by malaffoon** [comment]
Sounds good, thanks for the info.

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html (hunk 1 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: should we be using this everywhere, did you do a global search on this component?

Comment (esotrope) [reply webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: i'll run a search and see what i find

**Pull request merged**